### PR TITLE
Adds docs for CRI socket path change

### PR DESCRIPTION
**Pull request by ctauchen**
Rewrite of #479 

Product Version(s):

CC 3.16 and next
Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by ctauchen** [comment]
@Brian-McM

**Review by bmckercher123 — APPROVED** [review]
LGTM

File: calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx [hunk 1]
```diff
@@ -246,7 +246,26 @@ To change the CVSS threshold values, note the following:
 
 **Enable Runtime View**
 
-When Runtime View is enabled in Scan Settings, {{prodname}} actively monitors pods running in your Kubernetes clusters so it can correlate them with images you have recorded in the Image Assurance system. By default, this feature is disabled. If you disable the feature after it has been enabled, runtime information that is already collected for existing images will remain in the system.
+You can configure your scan settings to directly monitor pods that are running in your Kubernetes cluster.
+When enabled, the Runtime View service checks image information from the active container images in the cluster.
```
Comment (Brian-McM) [inline calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx]: I don't think we're calling it a "service" right now, the Runtime View is basically the combination of collecting information and displaying it in the UI. Could we just keep it as "Runtime View" and not "Runtime View service", as no "service" actually exists for it?

Comment (Brian-McM) [inline calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx]: "from the active container images in the cluster.", this is for all managed clusters, it's a global setting that is either on or off for all connected managed clusters.

**Review by Brian-McM — APPROVED** [review]

File: calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx]: Ok, changed.

Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx]: Changed.

**Pull request merged**